app/views/tournaments/index.html.erb

```diff
@@ -1,2 +1,6 @@
 <h1>Tournaments#index</h1>
 <p>Find me in app/views/tournaments/index.html.erb</p>
+
```

Comment:
Here please start creating just a div to wrap all the elements and add the UX actions